avaframe/com4FlowPy/flowClass.py

```diff
@@ -171,7 +172,18 @@ def calc_z_delta(self):
 
         if self.forestBool:
             if self.forestModule == "forestFrictionLayer":
-                _tanAlpha = self.tanAlphaFor
+                # default behavior - forest effect only neglected for start-cells
+                if self.nSkipForestCells == 1:
+                    if not self.is_start:
+                        _tanAlpha = self.tanAlphaFor
+                    else:
+                        _tanAlpha = self.tanAlpha
+                # forest effect also neglected for direct successors to the start-cell
+                elif self.nSkipForestCells == 2:
+                    if (not self.is_start) and (True not in [x.is_start for x in self.lOfParents]):
+                        _tanAlpha = self.tanAlphaFor
+                    else:
+                        _tanAlpha = self.tanAlpha
```

Comment:
I think this could be simplified (to make it more readable). :
The else is the same for both, and by combining the ifs...